src/intentManager.js

```diff
@@ -124,7 +137,6 @@ define([
                 };
             intents = dataTree.treeMap(getIntentMugs);
             if (intents.length > 0) {
-                xmlWriter.writeComment('Intents inserted by Vellum:');
```

Comment:
Removed because this was not actually writing a comment (`writeComment` is broken).